.rubocop.yml

```diff
@@ -1,2 +1,6 @@
 inherit_from:
-  - https://raw.githubusercontent.com/facebook/chef-cookbooks/main/.rubocop_55.yml
+  - https://raw.githubusercontent.com/facebook/chef-cookbooks/main/.rubocop.yml
```

Comment:
Should we add `GlobalStdStream` to the chef-cookbooks config that everything inherits from? No need for BM to be different, I don't think?

Reply:
I'd like it to be handled in chef-cookbooks eventually, but the general upgrade pattern we're going for is "bump version, exclude any AST-mutating lints that fire, ratchet code into better state, repeat." This is so that we can get on to the newer RuboCop/Cookstyle versions faster, as we want to automate the gem upgrades :-)
That said, the `required_ruby_version` bit shouldn't affect BM itself, so did that as low-hanging fruit (I did the same in https://github.com/facebook/grocery-delivery/pull/66/, and the same GlobalStdStream exclusion in https://github.com/facebook/chef-utils/commit/34f3bfa245398ecb4cbd27062c754d9eec2f879e